plugins/draco/module/vtkF3DGLTFReader.cxx

```diff
@@ -0,0 +1,14 @@
+#include "vtkF3DGLTFReader.h"
+
+#include "vtkF3DGLTFDocumentLoader.h"
+
+#include <vtkObjectFactory.h>
+
+//----------------------------------------------------------------------------
+vtkStandardNewMacro(vtkF3DGLTFReader);
+
+//----------------------------------------------------------------------------
+void vtkF3DGLTFReader::InitializeLoader()
+{
+  this->Loader = vtkSmartPointer<vtkF3DGLTFDocumentLoader>::New();
```

Comment:
I wonder if using a factory would be better. I know this is already merged in VTK, but it sounds exactly like the usecase for factory

Reply:
Interesting. The problem is we have to rely on some kind of static hacks like `init.cxx` but I can try.